recipes/xz_utils/all/conanfile.py

```diff
@@ -133,3 +133,5 @@ def package_info(self):
             self.cpp_info.defines.append("LZMA_API_STATIC")
         self.cpp_info.libs = tools.collect_libs(self)
         self.cpp_info.names["pkg_config"] = "liblzma"
+        self.cpp_info.names["cmake_find"] = "LibLZMA"
+        self.cpp_info.names["cmake_find_multi"] = "LibLZMA"
```

Comment:
```suggestion
        self.cpp_info.names["cmake_find_package"] = "LibLZMA"
        self.cpp_info.names["cmake_find_package_multi"] = "LibLZMA"
```

Reply:
Sorry, I can't see the error. Could you point me? https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info According docs, cmake_find_package and cmake_find_package_multi are correct.

Reply:
oh boy. forget about ...